src/System.Private.ServiceModel/src/System/ServiceModel/Channels/AddressingVersion.cs

```diff
@@ -32,6 +33,12 @@ public sealed class AddressingVersion
             Addressing10Strings.FaultAction, Addressing10Strings.DefaultFaultAction);
         private static MessagePartSpecification s_addressing10SignedMessageParts;
 
+        private static AddressingVersion s_addressing200408 = new AddressingVersion(Addressing200408Strings.Namespace,
+            XD.Addressing200408Dictionary.Namespace, Addressing200408ToStringFormat, Addressing200408SignedMessageParts,
+            Addressing200408Strings.Anonymous, XD.Addressing200408Dictionary.Anonymous, null,
+            Addressing200408Strings.FaultAction, Addressing200408Strings.DefaultFaultAction);
+        private static MessagePartSpecification s_addressing200408SignedMessageParts;
+
 
         private AddressingVersion(string ns, XmlDictionaryString dictionaryNs, string toStringFormat,
```

Comment:
Out code style guidelines require any blank line spacing to be only a single blank line.